centaur/src/main/resources/standardTestCases/preemptible_and_memory_retry.test

```diff
@@ -0,0 +1,24 @@
+name: preemptible_and_memory_retry
+testFormat: workflowfailure
+backends: [Papiv2]
+
+files {
+  workflow: retry_with_more_memory/preemptible_and_memory_retry.wdl
+  options: retry_with_more_memory/retry_with_more_memory.options
+}
+
+metadata {
+  workflowName: preemptible_and_memory_retry
+  status: Failed
+  "failures.0.message": "Workflow failed"
+  "failures.0.causedBy.0.message": "stderr for job `preemptible_and_memory_retry.imitate_oom_error_on_preemptible:NA:3` contained one of the `memory-retry-error-keys: [OutOfMemory,Killed]` specified in the Cromwell config. Job might have run out of memory."
```

Comment:
This is very cool as a workflow level error message!

Are there also call-level messages in the metadata for attempts 1 and 2?

Reply:
I think yes. My assumption is that call level `failures` key is also flattened the same way as other call level keys